CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to `@lmc-eu/spirit-design-system`
+
+First of all, thanks for your contribution to this project! ❤️ Here are some tips how to make your contributing efforts efficient and eventually accepted & merged.
+
+## General usage
+
+This project uses `makefile` for managing various chores, like dependency installation, testing, linting etc. Make sure you run `make` right after you clone the repository - it will set you up with everything needed to get started. Once you are done with that, have a look at what commands (targets) are available for you to run using `make` - `make install` etc.
```

Comment:
So we should list Make in development prerequisites in the main `README`, right? I can do that in #18.

Reply:
Yes, we should. Do it please in #18. Thanks.

Reply:
I've just found out there are a few more questions, I'll do it in a separate PR.